vite.config.js

```diff
@@ -8,6 +8,12 @@ export default defineConfig({
     metaversefilePlugin(),
     reactRefresh(),
   ],
+
+  optimizeDeps:{
+    entries: ['src/**/*.js', 'src/**/*.jsx','node_modules/**/*.js', '*.js', './**/*.jsx'],
```

Comment:
Can we whilelist the dirs instead of using a megaglob `**`, to make sure we are not captuing e.g. locally checked out dev apps and files?